DiscordChatExporter.Core/Exporting/MessageExporter.cs

```diff
@@ -30,6 +30,11 @@ private async ValueTask ResetWriterAsync(CancellationToken cancellationToken = d
         }
     }
 
+    // Unless explicitly called, file will be created before writing first message
+    public async ValueTask EnsureFileIsCreated(CancellationToken cancellationToken = default) {
+        await GetWriterAsync(cancellationToken);
```

Comment:
This would only result in a 0-byte file, right? I think the original intention was to create a fully structurally sound export, just with no messages.

Reply:
No, `GetWriterAsync` causes `WritePreambleAsync` to be run, which means you get the expected boilerplate as you said. Test coming up which should clear it up.

Reply:
Ah, cool